arch/arm/src/Makefile

```diff
@@ -165,7 +165,7 @@ $(KBIN): $(OBJS)
 	$(call ARCHIVE, $@, $(OBJS))
 
 board$(DELIM)libboard$(LIBEXT):
-	$(Q) $(MAKE) -C board TOPDIR="$(TOPDIR)" libboard$(LIBEXT) EXTRAFLAGS="$(EXTRAFLAGS)"
+	$(Q) $(MAKE) -C board TOPDIR="$(TOPDIR)" libboard$(LIBEXT) EXTRAFLAGS="$(KDEFINE) $(EXTRAFLAGS)"
```

Comment:
need for all other arch

Reply:
Can't test all other arch. Will open an Issue.

Reply:
This is a pure makefile change, so it's good after other arch change pass the build.

Reply:
Done. You told me to.

Reply:
Thanks!